images/kindnetd/go.mod

```diff
@@ -3,27 +3,64 @@ module sigs.k8s.io/kind/images/kindnetd
 go 1.13
 
 require (
-	github.com/godbus/dbus v4.1.0+incompatible // indirect
-	github.com/gogo/protobuf v1.2.1 // indirect
-	github.com/golang/protobuf v1.3.1 // indirect
-	github.com/google/gofuzz v1.0.0 // indirect
 	github.com/googleapis/gnostic v0.2.0 // indirect
-	github.com/json-iterator/go v1.1.6 // indirect
-	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
-	github.com/modern-go/reflect2 v1.0.1 // indirect
 	github.com/pkg/errors v0.8.1
+	github.com/spf13/pflag v1.0.5 // indirect
 	github.com/vishvananda/netlink v1.0.0
 	github.com/vishvananda/netns v0.0.0-20180720170159-13995c7128cc // indirect
-	golang.org/x/crypto v0.0.0-20190506204251-e1dfcc566284 // indirect
-	golang.org/x/net v0.0.0-20190509222800-a4d6f7feada5 // indirect
-	golang.org/x/oauth2 v0.0.0-20190402181905-9f3314589c9a // indirect
 	golang.org/x/time v0.0.0-20190308202827-9d24e82272b4 // indirect
 	gopkg.in/inf.v0 v0.9.1 // indirect
-	gopkg.in/yaml.v2 v2.2.2 // indirect
-	k8s.io/api v0.0.0-20190409021203-6e4e0e4f393b
-	k8s.io/apimachinery v0.0.0-20190404173353-6a84e37a896d
-	k8s.io/client-go v11.0.0+incompatible
-	k8s.io/kubernetes v1.14.2
-	k8s.io/utils v0.0.0-20190506122338-8fab8cb257d5
-	sigs.k8s.io/yaml v1.1.0 // indirect
+	k8s.io/api v0.0.0
+	k8s.io/apimachinery v0.0.0
+	k8s.io/client-go v0.0.0
```

Comment:
I really need your help here Ben, is this the best way to use kubernetes as a module?
we only need these

```
cmd/kindnetd/cni.go:    corev1 "k8s.io/api/core/v1"
cmd/kindnetd/cni.go:    "k8s.io/utils/net"
cmd/kindnetd/main.go:   corev1 "k8s.io/api/core/v1"
cmd/kindnetd/main.go:   metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
cmd/kindnetd/main.go:   "k8s.io/client-go/kubernetes"
cmd/kindnetd/main.go:   "k8s.io/client-go/rest"
cmd/kindnetd/main.go:   "k8s.io/utils/net"
cmd/kindnetd/masq.go:   utildbus "k8s.io/kubernetes/pkg/util/dbus"
cmd/kindnetd/masq.go:   utiliptables "k8s.io/kubernetes/pkg/util/iptables"
cmd/kindnetd/masq.go:   utilexec "k8s.io/utils/exec"
```

Reply:
Not like this ...
Will follow up Monday but ... Why are we even touching this? This is totally unrelated to not panicing.

Reply:
the `pkg/util/iptables` has changed in the last releases, I was especially interested in the `dbus` removal, but I just found it didn't make it to the 1.16 and is only available in 1.17

https://github.com/kubernetes/kubernetes/pull/81517

I'm experimenting with several approaches, basically trying to understand if we can simplify the ipmasq code.

Reply:
we can use go-iptables and not rely on kubernetes for this